workflow.rst

```diff
@@ -90,35 +90,28 @@ like this:
             <framework:config>
                 <framework:workflow name="blog_publishing" type="workflow">
                     <framework:audit-trail enabled="true"/>
-
                     <framework:marking-store type="single_state">
                         <framework:argument>currentPlace</framework:argument>
                     </framework:marking-store>
-
                     <framework:support>App\Entity\BlogPost</framework:support>
-
+                    <framework:initial_marking>draft</framework:initial_marking>
```

Comment:
@OskarStark really unsure of the pattern, I never use xml :/
Also i remove spaces like php and yml docbloc

Reply:
I am unsure too about the xml config

cc @lyrixx

Reply:
friendly ping @lyrixx can you please give us some feedback here?